fem/doftoquad.hpp

```diff
@@ -0,0 +1,67 @@
+// Copyright (c) 2010, Lawrence Livermore National Security, LLC. Produced at
+// the Lawrence Livermore National Laboratory. LLNL-CODE-443211. All Rights
+// reserved. See file COPYRIGHT for details.
+//
+// This file is part of the MFEM library. For more information and source code
+// availability see http://mfem.org.
+//
+// MFEM is free software; you can redistribute it and/or modify it under the
+// terms of the GNU Lesser General Public License (as published by the Free
+// Software Foundation) version 2.1 dated February 1999.
+
+#ifndef MFEM_BILIN_DQM_HPP
+#define MFEM_BILIN_DQM_HPP
+
+#include "../config/config.hpp"
+#include "../general/karray.hpp"
+
+namespace mfem
+{
+
+// ***************************************************************************
+// * kDofQuadMaps
+// ***************************************************************************
+class kDofQuadMaps
```

Comment:
If this file is called `doftoquad.hpp`, why is the class called `kDofQuadMaps`? We should make sure to address this before merging.